WMIAdventure/frontend/src/js/components/battle/atoms/MiniCardView/MiniCardView.js

```diff
@@ -5,6 +5,16 @@ import LevelBorder from "./styled-components/LevelBorder";
 import iconPlaceholder from '../../../../../assets/icons/upload_image_dark.svg';
 
 class MiniCardView extends React.Component {
+    /*
+    props:
+        setTranslateX -> handle init animation
+        enemy -> signal that card belong to enemy
+        user -> signal that card belong to current user
+        rank -> information about card level
+        visible -> handle visibility of card
+        icon -> card icon
+     */
+

```

Comment:
Super, jako że nie mamy TypeScripta to można zawsze starać się pisać takie komentarze, to bardzo ułatwia pracę, nie trzeba skakać po plikach